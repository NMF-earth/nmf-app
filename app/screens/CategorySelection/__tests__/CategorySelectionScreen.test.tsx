import React from "react";
import { create } from "react-test-renderer";

import CategorySelectionScreen from "../CategorySelectionScreen";

it("CategorySelectionScreen renders correctly", () => {
  const tree = create(<CategorySelectionScreen />).toJSON();
  expect(tree).toMatchSnapshot();
});
