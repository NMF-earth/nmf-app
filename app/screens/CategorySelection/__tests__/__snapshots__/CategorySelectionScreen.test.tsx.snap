// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CategorySelectionScreen renders correctly 1`] = `
<RCTScrollView
  style={
    {
      "backgroundColor": "#FFFFFF",
      "flex": 1,
      "paddingHorizontal": 16,
    }
  }
>
  <View>
    <Text.H2
      style={
        {
          "paddingBottom": 16,
          "paddingTop": 20,
        }
      }
    >
      Select a category
    </Text.H2>
    <Tag
      icon="restaurant"
      onPress={[Function]}
      text="Meal"
    />
    <Tag
      icon="airplane"
      onPress={[Function]}
      text="Transport"
    />
    <Tag
      icon="barcode-sharp"
      onPress={[Function]}
      text="Scan product"
    />
    <Tag
      icon="play-circle"
      onPress={[Function]}
      text="Streaming"
    />
    <Tag
      icon="card"
      onPress={[Function]}
      text="Purchase"
    />
    <Tag
      icon="shirt"
      onPress={[Function]}
      text="Fashion"
    />
    <Tag
      icon="nutrition"
      onPress={[Function]}
      text="Food"
    />
    <Tag
      icon="flash"
      onPress={[Function]}
      text="Electricity"
    />
    <Tag
      icon="build"
      onPress={[Function]}
      text="Custom"
    />
    <View
      style={
        {
          "height": 10,
        }
      }
    />
  </View>
</RCTScrollView>
`;
